automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPyKernelSpec.scala

```diff
@@ -51,14 +88,13 @@ class NotebookPyKernelSpec extends ClusterFixtureSpec with NotebookTestUtils {
       contentSecurityHeader.get.value should include ("https://bvdp-saturn-prod.appspot.com")
     }
 
-    // we have to disable SSL validation for BigQuery to work on the command line. this is not ideal, so should be resolved as soon as possible
```

Comment:
I _think_ the SSL validation was fixed, but wasn't sure. Will see how the test does :)